docs/README.md

```diff
@@ -0,0 +1,229 @@
+# Indexer API Ddocs
+
+<!-- no toc -->
+- [Endpoints](#endpoints)
+- [Response Format](response-format.md)
+- [Pagination](pagination.md)
+- [Rate Limiting](#rate-limiting)
+
+## Endpoints
+
+#### GET /chains
+
+Returns a list of supported chains and features. `chainId` is the unique identifier for each chain and is used in other endpoints. While chain id returned is a number, endpoints can also accept chain id as hex string (e.g. `0x1e` instead of `30` for Rootstock).
+
+Example response:
+```json
+{
+  "data": [
+    {
+      "chainId": 30,
+      "name": "Rootstock",
+      "features": [
+        "sdex"
+      ]
+    }
+  ],
+  "timestamp": 1630000000
+}
+```
+
+#### GET /tokens
+
+Returns a list of supported tokens and their details. Supports `pagination`. 
+Tokens are ordered by `address`, so native tokens are listed first.
+
+Endpoint data is cached for 1 minute.
+
+`GET /tokens?chainId=30&spam=1`
+
+Use optional `chainId` query parameter to filter tokens by chain.
+Use optional `spam=1` query parameter to include tokens marked as spam / ignored by our dex.
+
+Example response:
+```json
+{
+  "data": [
+    {
+      "symbol": "RBTC",
+      "name": "Rootstock Smart Bitcoin",
+      "decimals": 18,
+      "chainId": 30,
+      "address": "0x0000000000000000000000000000000000000000",
+      "usdPrice": "64499.13340664030853125",
+      "usdPriceDate": "2024-08-26 08:17:00"
+    }
+  ],
+  "timestamp": 1630000000
+}
+```
+
+
+### Legacy endpoints
+
+Legacy endpoints are endpoints moved from the old API and are mostly used for rootstock chain data. It will likely be deprecated in the future. They are prefixed with `/legacy`.
+
+#### GET /legacy/amm
+
+Returns list of rootstock AMM pools. `chainId` is required.
```

Comment:
```suggestion
Returns list of Rootstock AMM pools. `chainId` is required.
```

Reply:
After reading the whole file, I think that the correct adjustment would be to omit `Rootstock` from descriptions of the legacy endpoints since it's mentioned at the beginning that they are used mostly for Rootstock chain data.

WDYT @soulBit?